README.md

```diff
@@ -235,58 +235,69 @@ For headaches, take Ibuprofen
 The gobin command installs/runs main packages.
 
 Usage:
-	gobin [-m] [-r|-p] [-n|-g] packages [run arguments...]
+	gobin [-m] [-run|-p] [-nonet|-u] packages [run arguments...]
 
-The packages argument to gobin is similar to that of the go tool (in module
-mode) with the additional constraint that the list of packages must be main
-packages.
+The packages argument to gobin is similar to that of the go get command (in
```

Comment:
How about starting with an executive summary?

     The gobin command builds, installs, and possibly runs an executable binary for each of the named packages, which must all be main packages.

?

Reply:
Good idea. Done